### PR TITLE
[#63]: Add compatibility table

### PR DESCRIPTION
**Pull request by aprasolova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,13 @@ This role deploys [Storage node][neofs-node] for [NeoFS][neofs].
 
 [neofs-node]: https://github.com/nspcc-dev/neofs-node
 [neofs]:      https://fs.neo.org
+
+### NeoFS Storage Compatibility Table
+
+| NeoFS Node version | ansible-neofs-storage role version | min NeoFS Node supported version | max NeoFS Node compatible version |
+|--------------------|------------------------------------|----------------------------------|-----------------------------------|
+| 0.30.2             | v0.0.33                            | 0.30.0                           | 0.30.2                            |
```
Comment (roman-khimov) [inline README.md:13]: So 0.36.0 node is not compatible with 0.0.33 role? How does it work then?

File: README.md [hunk 2]
```diff
@@ -5,3 +5,13 @@ This role deploys [Storage node][neofs-node] for [NeoFS][neofs].
 
 [neofs-node]: https://github.com/nspcc-dev/neofs-node
 [neofs]:      https://fs.neo.org
+
+### NeoFS Storage Compatibility Table
+
+| NeoFS Node version | ansible-neofs-storage role version | min NeoFS Node supported version | max NeoFS Node compatible version |
```
Comment (roman-khimov) [inline README.md:11]: Why do we have _three_ NeoFS node versions?

Comment (aprasolova) [reply README.md:11]: A given role tag introduces the updates for the "NeoFS Node version", "min" and "max" versions correspond to minimal supported and maximum compatible versions for this tag.

File: README.md (hunk 1 above)
Comment (aprasolova) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (roman-khimov) [reply README.md:11]: OK, let's have it this way for now. Can be fixed later if any better ideas appear.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**